cf_units/__init__.py

```diff
@@ -359,14 +359,14 @@ def julian_day2date(julian_day, calendar):
 
         >>> import cf_units
         >>> import datetime
-        >>> dt = datetime.datetime(1970, 1, 1, 0, 0, 0)
+        >>> dt = cftime.datetime(1970, 1, 1, 0, 0, 0)
         >>> jd = cf_units.date2julian_day(dt, cf_units.CALENDAR_STANDARD)
         >>> print(cf_units.julian_day2date(jd, cf_units.CALENDAR_STANDARD))
         1970-01-01 00:00:00
 
     """
 
-    return cftime.DateFromJulianDay(julian_day, calendar)
+    return cftime.datetime.fromordinal(julian_day, calendar)
```

Comment:
Have just realised that this docstring is wrong (and possibly has been for some time) as "real" `datetime.datetime` instances are not returned even with `cftime.DateFromJulianDay` under cftime v1.2.1.

Given that these julian day functions just pass their args straight to the relevant `cftime` functions, I'm wondering if there is any real benefit in maintaining them and trying to keep the docstrings up-to-date with `cftime` changes.  Could we just remove them and direct users to use the `cftime` functions directly instead?